### ISSUE TITLE
Invalid Turnstile Captcha secret still works

### ISSUE THREAD

**Issue by RMartinOscar** [pelican-dev/panel#792]
This is due to [coderflexx/filament-turnstile](<https://github.com/search?q=repo%3Acoderflexx%2Ffilament-turnstile+secret&type=code>) & [react-turnstile](<https://github.com/search?q=repo%3ALe0Developer%2Freact-turnstile%20secret&type=code>) not enforcing [cloudflare server-side-validation](https://developers.cloudflare.com/turnstile/get-started/server-side-validation/)

**Comment by Boy132** [comment]
The filemant turnstile does: https://github.com/coderflexx/laravel-turnstile/blob/main/src/LaravelTurnstile.php

**Comment by RMartinOscar** [comment]
Then there's something else cause 
![image](https://github.com/user-attachments/assets/515230bd-3662-48a5-875f-8ae906136795)
![image](https://github.com/user-attachments/assets/7709506a-0b35-4b0b-942d-98c6b109ddea)
_Regenerated since_

**Comment by Boy132** [comment]
> Then there's something else cause ![image](https://private-user-images.githubusercontent.com/40749467/393630432-515230bd-3662-48a5-875f-8ae906136795.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM2ODIwMDYsIm5iZiI6MTczMzY4MTcwNiwicGF0aCI6Ii80MDc0OTQ2Ny8zOTM2MzA0MzItNTE1MjMwYmQtMzY2Mi00OGE1LTg3NWYtOGFlOTA2MTM2Nzk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA4VDE4MTUwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0YWJkNzY1MzcyNjczMGVlZDdkYWIyZWVlNjUxMmJmMDM3NTdhMmY5ZDIyMDJhMDY2OGJlZjM4YzY1ZTYwMGEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ZQHEaTDbR6CLCx4R0lbmo5V16PNqkEbMJrIz6PyYJbw) ![image](https://private-user-images.githubusercontent.com/40749467/393630464-7709506a-0b35-4b0b-942d-98c6b109ddea.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM2ODIwMDYsIm5iZiI6MTczMzY4MTcwNiwicGF0aCI6Ii80MDc0OTQ2Ny8zOTM2MzA0NjQtNzcwOTUwNmEtMGIzNS00YjBiLTk0MmQtOThjNmIxMDlkZGVhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA4VDE4MTUwNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFlNjZlZTlmMjcwNTA3NmFhNzI5ZmE0MTdlZWJkZmI0OWU2Njc3OGZhOWI3YmMwNmQ4YTAzZGVlMjI5MjAxOGYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.8IXUryjsuzIQNMykf4Cm1Z2w8a49eoQApRtW3ECTHHI) _Regenerated since_

And it lets you login when clicking "Sign in"?

**Comment by RMartinOscar** [comment]
yup it just skips the validation, but if you enter invalid sitekey then it errors out

**Comment by Boy132** [comment]
Can't reproduce that:
![image](https://github.com/user-attachments/assets/323ff532-2762-49e7-8f6a-07956944b709)
![image](https://github.com/user-attachments/assets/5c947a05-6bc8-49ed-badb-554e941ab051)

**Comment by RMartinOscar** [comment]
Not needed now that we removed the old client area
